staging/src/k8s.io/pod-security-admission/metrics/metrics.go

```diff
@@ -98,15 +98,27 @@ func (r *PrometheusRecorder) RecordEvaluation(decision Decision, policy api.Leve
 		}
 	}
 
-	r.evaluationsCounter.CachedInc(evaluationsLabels{
-		decision:    string(decision),
-		level:       string(policy.Level),
-		version:     version,
-		mode:        string(evalMode),
-		operation:   operationLabel(attrs.GetOperation()),
-		resource:    resourceLabel(attrs.GetResource()),
-		subresource: attrs.GetSubresource(),
-	})
+	// prevent cardinality explosion by only recording the platform namespaces
+	namespace := attrs.GetNamespace()
+	if !(namespace == "openshift" ||
+		strings.HasPrefix(namespace, "openshift-") ||
+		strings.HasPrefix(namespace, "kube-") ||
+		namespace == "default") {
+		namespace = "non-platform"
```

Comment:
it might not be obvious for admins that "non-platform" is a sentinel value. What if the `namespace` variable is set back to the empty string in this case?

Reply:
the PR merged before I noticed the comment. This is getting fixed in https://github.com/openshift/kubernetes/pull/1489